{"metadata":{"role":"collectionGroup","modules":[{"name":"PingOneWallet"}],"roleHeading":"API Collection","title":"LocalizedError Implementations"},"sections":[],"kind":"article","variants":[{"paths":["\/documentation\/pingonewallet\/walletexception\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/LocalizedError-Implementations","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"topicSections":[{"generated":true,"identifiers":["doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/failureReason","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/helpAnchor","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/recoverySuggestion"],"title":"Instance Properties"}],"references":{"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException/recoverySuggestion":{"kind":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"abstract":[],"type":"topic","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/recoverySuggestion","url":"\/documentation\/pingonewallet\/walletexception\/recoverysuggestion","role":"symbol","title":"recoverySuggestion"},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet":{"kind":"symbol","type":"topic","role":"collection","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet","url":"\/documentation\/pingonewallet","abstract":[{"text":"This defines the usage and interfaces of the PingOne Wallet SDK for iOS to interact with PingOne credentialing services. The SDK provides for a user to receive, save, and share credentials for identification and verification purposes.","type":"text"}],"title":"PingOneWallet"},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException":{"role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"WalletException","kind":"identifier"}],"kind":"symbol","type":"topic","title":"WalletException","navigatorTitle":[{"text":"WalletException","kind":"identifier"}],"abstract":[],"url":"\/documentation\/pingonewallet\/walletexception","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException"},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException/failureReason":{"role":"symbol","kind":"symbol","title":"failureReason","type":"topic","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/failureReason","abstract":[],"url":"\/documentation\/pingonewallet\/walletexception\/failurereason","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failureReason","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}]},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException/helpAnchor":{"kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[],"type":"topic","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException\/helpAnchor","url":"\/documentation\/pingonewallet\/walletexception\/helpanchor","role":"symbol","title":"helpAnchor"}}}